addons/mail/static/src/messaging/component/followers_menu/followers_menu.js

```diff
@@ -0,0 +1,167 @@
+odoo.define('mail.messaging.component.FollowersMenu', function (require) {
+'use strict';
+
+const components = {
+    Follower: require('mail.messaging.component.Follower'),
+};
+const useStore = require('mail.messaging.component_hook.useStore');
+
+const { Component } = owl;
+const { useRef, useState } = owl.hooks;
+
+class FollowersMenu extends Component {
```

Comment:
I don't think we want plural in names.

Reply:
Yeah but it's a menu of followers, not a single follower ... what do you think ?

Reply:
We can see it as "FollowerMenu" is a menu for the "Follower" feature.

If the "multi" aspect is important, I would name it FollowerList instead similar do what we do for other lists.

Reply:
I'm good with either `FollowersMenu` or `FollowerListMenu`, but not `FollowerMenu` because this is misleading. `MessagingMenu` is good because "messaging" is a wide notion, but here "follower" is well defined and corresponds to a single entry of this menu.

Reply:
Which one do you prefer @seb-odoo ? :)

Reply:
No plural, so FollowerListMenu